src/Content/AttachmentLocal.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Phlib\Mail\Content;
+
+use Phlib\Mail\Exception\RuntimeException;
+
+/**
+ * Attachment class used to add local files to a Mail object
+ *
+ * @package Phlib\Mail
+ */
+class AttachmentLocal extends Attachment
```

Comment:
Other than the concern about large files and memory issues. I think this should be fine for now.